tests/foreman/cli/test_hostgroup.py

```diff
@@ -129,13 +132,43 @@ def test_positive_create_with_puppet_proxy(self):
         @Assert: Hostgroup is created and has puppet proxy server assigned
 
         """
-        puppet_proxy = Proxy.list()[0]
+        puppet_proxy = Proxy.list({
+            'search': 'url = https://{0}:9090'.format(settings.server.hostname)
+        })[0]
         hostgroup = make_hostgroup({'puppet-proxy': puppet_proxy['name']})
         self.assertEqual(
             puppet_proxy['id'],
             hostgroup['puppet-master-proxy-id'],
         )
 
+    @run_only_on('sat')
+    @tier1
+    def test_positive_create_with_architecture(self):
+        """Check if hostgroup with architecture can be created
+
+        @Feature: Hostgroup - Positive create
+
+        @Assert: Hostgroup should be created and has architecture assigned
+
+        @BZ: 1354544
```

Comment:
This bug was opened for 6.2.0 and not affecting previous versions. Test has passed on 6.1.8